src/commonNative/kotlin/com/epam/drill/agent/AgentArguments.kt

```diff
@@ -14,7 +14,8 @@ data class AgentArguments(
     val groupId: String = "",
     val logLevel: String = LogLevel.ERROR.name,
     val logFile: String? = null,
-    val webAppNames: String = ""
+    val webAppNames: String = "",
+    val isAsyncApp: Boolean = false
```

Comment:
we should use only string constans. "false" or "true" cuzz agent arguments just a string.

Reply:
fix it with https://gist.github.com/36bd4f3525495dbc3f483bacb4ff8475